tests/BuildScriptGenerator.Tests/DefaultBuildScriptGeneratorTest.cs

```diff
@@ -635,7 +635,8 @@ public void Checkers_DontFailTheBuild_WhenTheyThrow()
                     Options.Create(commonOptions)),
                 checkers,
                 NullLogger<DefaultBuildScriptGenerator>.Instance,
-                new DefaultStandardOutputWriter());
+                new DefaultStandardOutputWriter(),
+                null);
```

Comment:
We'll run into issues if we provide `null` wherever a method needs `TelemetryClient` -- it's probably easiest to create a `TestTelemetryClient` class in the `Oryx.Tests.Common` project that inherits from `TelemetryClient` and then we override its logging methods that we use throughout the repo (specifically within the methods of the extensions class) to do nothing so that we're not passing in a `null` object, but we don't actually do anything with the client.

Reply:
We are unable to clear a test class that inherits from TelemetryClient because TelemetryClient is a sealed class.
I created a mock class that returns a telemetry client object instead.